changelog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## NEXT
+
+- Fixed a bug where metadata on floating nodes would not be correctly carried over to the minimal graph when calling
+  `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph.get_minimal_graph`. This also fixes an issue where
+  `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph.is_minimal_graph` could return `False` on minimal
+  graphs that contain floating nodes.
```

Comment:
By definition is the graph minimal if it has floating nodes? @matteo-causalens what do you think?

Reply:
yes, we converged to this if I remember well.